Unio/Computed.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  Computed.swift
+//  Unio
+//
+//  Created by marty-suzuki on 2020/2/19.
+//  Copyright © 2020 tv.abema. All rights reserved.
+//
+
+import Foundation
+
+/// Represents computed property
+public final class Computed<Element> {
+
+    public var value: Element {
+        return getter()
```

Comment:
Possibly the best place to use `@inlinable`?

Reply:
Fixed at cd5d73be568f892ae13788fa8d03ae9e44138d6f

Reply:
👍